asn1/GenericPacket.asn

```diff
@@ -21,12 +21,44 @@ IMPORTS
     FROM InterledgerQuotingProtocol
 ;
 
-PACKET ::= CLASS {
+SideProtocolData ::= SEQUENCE OF SEQUENCE {
+  protocolName IA5String,
+  protocolData OCTET STRING
+}
+
+CustomRequest ::= SEQUENCE {
+  -- Additional data for protocol extensibility
+  sideProtocolData SideProtocolData
+}
+
+CustomResponse ::= SEQUENCE {
+  -- Additional data for protocol extensibility
+  sideProtocolData SideProtocolData
+}
+
+HIGHPACKET ::= CLASS {
+    &typeId UInt16 UNIQUE,
+    &Type
+} WITH SYNTAX {&typeId &Type}
+
+HighPacketSet HIGHPACKET ::= {
+    -- nothing here yet, will start filling up when the LowPacketSet is full
+}
+
+-- wraps a high packet into a low packet:
+HighPacketWrapper ::= SEQUENCE {
```

Comment:
How are you proposing that this be used? (You don't have type ids defined for the LLL)